plugins/idea/pluginBase/src/main/java/com/imaginea/kodebeagle/base/tasks/QueryKBServerTask.java

```diff
@@ -80,8 +78,7 @@
     private String httpErrorMsg;
     private WindowObjects windowObjects = WindowObjects.getInstance();
     private ProjectTree projectTree = new ProjectTree();
-    private ESUtils esUtils = new ESUtils();
-    private JSONUtils jsonUtils = new JSONUtils();
+    private SearchUtils searchUtils = new SearchUtils();
     private List<CodeInfo> spotlightPaneTinyEditorsInfoList = new ArrayList<CodeInfo>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2293)